### PR TITLE
EZQMS-279: Remove .ProseMirror global css

### PR DESCRIPTION
**Pull request by annano**
# Contribution checklist

## Checklist

* [ ] - UI test added to added/changed functionality?
* [ ] - Screenshot is added to PR if applicable ?
* [ ] - Does the code work? Check whether function and logic are correct.
* [ ] - Does Changelog.md is updated with changes?
* [ ] - Does the translations are up to date?
* [ ] - Does it well tested?
* [ ] - Tested for Chrome.
* [ ] - Tested for Safari.
* [ ] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [ ] - Rebase your branch onto master and upstream branch
* [ ] - Is there any redundant or duplicate code?
* [ ] - Are required links are linked to PR?
* [ ] - Does new code is well documented ?

## Related issues

A list of closed updated issues

### CODE REVIEW

File: plugins/gmail-resources/src/components/NewMessage.svelte [hunk 1]
```diff
@@ -245,12 +245,6 @@
     border-radius: 0.25rem;
     caret-color: var(--theme-caret-color);
 
-    :global(.ProseMirror) {
```
Comment (annano) [inline plugins/gmail-resources/src/components/NewMessage.svelte:248]: @BykhovDenis please review these changes, as I do not know where this is used.

The css should be added as `text-editor-view_full-height` class.

File: packages/theme/styles/prose.scss [hunk 2]
```diff
@@ -75,3 +75,20 @@ pre.proseCodeBlock > code.proseCode {
 }
 
 .proseHeading { line-height: 110%; }
+
+.proseH1 {
```
Comment (annano) [inline packages/theme/styles/prose.scss]: @SasLord is this css used anywhere? I have not found it being used.

Can we remove it?

Comment (SasLord) [reply packages/theme/styles/prose.scss]: proseHeading указан в extensions.ts, он ко всем заголовкам подставляется. Иначе их сильно разносит, если заголовок не в одну строку.

Comment (annano) [reply packages/theme/styles/prose.scss]: I mean `proseH1` and `proseH2`

Comment (annano) [reply packages/theme/styles/prose.scss]: As discussed offline, removed

**Review by aonnikov — APPROVED** [review]

**Pull request merged**